packages/generator-langium/templates/gitignore.txt

```diff
@@ -4,5 +4,5 @@
 !.vscode/tasks.json
 node_modules/
 out/
-src/language/generated/
+src/generated/
```

Comment:
It's not working as expected:

> For example, a pattern `doc/frotz/` matches `doc/frotz` directory, but not `a/doc/frotz` directory; however `frotz/` matches `frotz` and `a/frotz` that is a directory (all paths are relative from the `.gitignore` file).
>
> from https://git-scm.com/docs/gitignore#_pattern_format

Maybe change it into `packages/language/src/generated/` or `generated/`.

---

Consider ignoring `*.tsbuildinfo` since it's getting generated in the build process.

Reply:
I haven't checked the generated project outside langium repo. That shadows some of your findings. I will fix this. Thank you.

Reply:
Done ⬇️